Extend/Model/Order.php

```diff
@@ -228,4 +246,17 @@ public function getTranslatedState()
     {
         return self::getTranslatedStates()[$this->oxorder__wdoxidee_orderstate->value] ?? '';
     }
+
+    /**
+     * Returns the full name of the payment method
+     *
+     * @return string
+     */
+    public function getPaymentMethodName()
```

Comment:
There already is method getOrderPayment. I think you can call that method in the template directly and access its description.